README.md

```diff
@@ -1160,11 +1223,8 @@ Create the `run_multimer_jobs_SLURM.sh` script and place the following code in i
 #Adjust this depending on the node
 #SBATCH --ntasks=8
 #SBATCH --mem=64000
-
-module load Anaconda3 
```

Comment:
But sth has to be loaded, maybe Mamba?

Reply:
don't think so, e.g. for the tests I do simply this:
https://github.com/KosinskiLab/AlphaPulldown/blob/main/test/test_predictions_slurm.py#L74-L93

Reply:
On the cluster, at least ours, conda is not installed by default, so they would not have conda command (maybe you installed it globally on your account). Many users incl. me don't want to have such things installed globally to avoid conflicting dependencies and load them on demand through virtual envs, modules etc. 

In the doc, I would write all slurm examples as matched for EMBL cluster so 1) EMBL users can just copy 2) other users have an example. I currently use Mamba successfully for AlphaPulldown (module load Mamba).